### PR TITLE
Usage documentation

### PR DESCRIPTION
**Pull request by BereGabor**
md documentations

### CODE REVIEW

File: export_repos.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    print('Export git repos!')
+    print('Expected root folder structure:')
+    print('root -> contain the projects')
+    print('   |- project -> contain the repos of the project')
+    print('      |-  gitrepo -> git repo')
+    print('')
+    print('Result generated in output folder as source folders structured')
+    print('project folder name will be the project_name option in getstats cli command')
+
+PYTHON = "/usr/local/bin/python3.7"
```
Comment (vifactor) [inline export_repos.py]: This looks very specific to your environment, please, do not hard-code

File: README.md [hunk 2]
```diff
@@ -6,3 +6,85 @@ The idea is to modernize the existing tool:
  - replace self-made calls to git with calls to either PythonGit or pygit2 libraries
  - embed good-looking (bokeh?, gnuplot html?) graphs instead of gnuplot ones
  - add author contribution plots
+ 
+ Requirements:
+ - pyton 3
```
Comment (vifactor) [inline README.md]: misprint

File: README.md [hunk 3]
```diff
@@ -6,3 +6,85 @@ The idea is to modernize the existing tool:
...
+## export_repos Usage
+export_repos.py is a tool script to export well structured git repos.  
+Requiered folder structure:  
+
+* root  
+  * project1  
+    * GitRepo  
+    * GitRepo1  
+    * GitRepon  
+  * projet2
```
Comment (vifactor) [inline README.md]: projeCt2

File: README.md [hunk 4]
```diff
@@ -6,3 +6,85 @@ The idea is to modernize the existing tool:
...
+* root  
+  * project1  
+    * GitRepo  
+    * GitRepo1  
+    * GitRepon  
+  * projet2  
+    * p2repo1  
+    * p2repo2  
+
+`pyton export_repos.py [root-folder] [output-folder]`
```
Comment (vifactor) [inline README.md]: pytHon

File: export_repos.py (hunk 1 above)
Comment (vifactor) [inline export_repos.py]: Next line are a hardcoded strings for your specific environment. Need change.

**Comment by BereGabor** [comment]
I made requested changes.

**Comment by BereGabor** [comment]
I submit a new pullrequest with Squashed commit contained only md file changes

**Pull request closed without merging**